docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -123,10 +125,12 @@ Otherwise, it queries for an exact match and only matches that specific path:
 
 [source,properties]
 ----
-quarkus.http.auth.permission.permit1.paths=/public/*,/css/*,/js/*,/robots.txt
+quarkus.http.auth.permission.permit1.paths=/public*,/css/*,/js/*,/robots.txt    <1>
 quarkus.http.auth.permission.permit1.policy=permit
 quarkus.http.auth.permission.permit1.methods=GET,HEAD
 ----
+<1> The path pattern `/public*` matches request paths `/public`, `/public/` and all other sub-paths like `/public/info`,
+but will never match the `/public-info` path. In fact, Quarkus treats this pattern exactly as it treats `/public/*`.
```

Comment:
I got confused when I read the first time, why exactly `/public-info`, I believe your point is, `*` at the end of the expression matches multiple path segments but not any word starting from `/public`, which is what I propose to highlight

Reply:
If `*` is in the middle, `/a/*/d` then it matches a single path segment, right ? I.e, `/a/b/d` works but `/a/b/c/d` does not ?

Reply:
yes and yes

indeed I tried to make it clear that /public* won't match other word like /public-info because it got me confused previously, I expect it to work

will try to rewrite this

Reply:
Fixed.